src/content/en/2024/performance.md

```diff
@@ -226,11 +226,26 @@ Typically, LCP element rendering takes a long time if the LCP element hasn't bee
 
 It's interesting to observe the different LCP challenges that websites across various datasets face. While an average website from the CrUX dataset struggles with image load delay, websites from the RUMvision dataset often face rendering delay issues. Nevertheless, all websites can benefit from using performance monitoring tools with Real User Monitoring (RUM), as these tools provide deeper insights into the performance issues experienced by real users.
 
-#### LCP lazy-loading
+#### LCP static discoverability
+
+One of the most effective ways to optimize the LCP resource load delay is to ensure the resource can be discovered as early as possible. If you make the resource discoverable in the initial HTML document, it enables the LCP resource to begin downloading sooner.
+
+{{ figure_markup(
+  caption="The percent of mobile pages on which the LCP element was not statically discoverable.",
+  content="35%",
+  classes="big-number",
+  sheets_gid="200850285",
+  sql_file="lcp_preload_discoverable.sql"
+)
+}}
 
-One of the ways to optimize the LCP resource load delay is to ensure the resource can be discovered as early as possible. If you make the resource discoverable in the initial HTML document, it enables the LCP resource to begin downloading sooner. A big obstacle to LCP resource discoverability is lazy loading of the LCP resource.
+Unfortunately, 35% of mobile websites do not have an LCP element that is statically discoverable in the document. While this is a slight improvement over the 39% we saw in 2022, it's still a significant blocker of LCP performance.
 
-Overall, lazy-loading images is a helpful performance technique that should be used to postpone loading of non-critical resources until they are near the viewport. However, using lazy-loading on the LCP image will delay the browser from loading it quickly. That is why lazy-loading should not be used on LCP elements. In this section, we explore how many sites use this performance anti-pattern.
+As we'll explore in the following sections, there are three primary ways that websites prevent their LCP resources from being statically discoverable: lazy-loading, CSS background images, and client-side rendering.
```

Comment:
Nit: lazy loading not lazy-loading. Lazy-loading would be used as an adjective, such as "a lazy-loaded image".

(Barry has taught me well 😅)